examples/runDecamQuickTest.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+print_error() {
+    >&2 echo "$@"
+}
+
+PRODUCT_DIR=${VALIDATE_DRP_DIR}
+# OS X El Capitan SIP swallows DYLD_LIBRARY_PATH so export the duplicate in LSST_LIBRARY_PATH
+if [[ -z "$DYLD_LIBRARY_PATH" ]]; then
+    export DYLD_LIBRARY_PATH=$LSST_LIBRARY_PATH
+fi
+
+CAMERA=DecamQuick
+CONFIG_FILE="${PRODUCT_DIR}"/config/decamConfig.py
+MAPPER=lsst.obs.decam.DecamMapper
+
+"${PRODUCT_DIR}"/examples/runExample.sh $CAMERA $MAPPER ${VALIDATION_DATA_DECAM_DIR}/instcal "${CONFIG_FILE}" ingestImagesDecam.py processCcdDecam.py
```

Comment:
Line too long.

Minor nit: setting `${PRODUCT_DIR}` based on `${VALIDATE_DRP_DIR}` saved you a couple of characters here, but I don't really think it was worth it.